ui/src/app/edge/index/energymonitor/chart/section/gridsection.component.ts

```diff
@@ -8,12 +8,13 @@ import { AbstractSection, SvgSquarePosition, SvgSquare, EnergyFlow, SvgEnergyFlo
     templateUrl: './section.component.html'
 })
 export class GridSectionComponent extends AbstractSection {
-
+    private GridMode: number;
```

Comment:
Variables should start with a lower case letter. As mentioned above: create a enum type to keep the gridMode instead of using a plain number.
https://www.typescriptlang.org/docs/handbook/enums.html